docs/build/get-started.mdx

```diff
@@ -671,6 +493,188 @@ After `fluence build`, re-run `cargo test --workspace` and voila, all is well ag
 
 Now, we're ready to talk about Deal deployment and Compute Marketplace.
```

Comment:
Aaand it doesn't talk about that any more. 
```suggestion
Now, before we're ready to move on to Deployment to Marketplace, let's go through Wallet setup and get tokens necessary for the Deployments.
```

And move `Now, we're ready to talk about Deal deployment and Compute Marketplace.` to a later point.

Reply:
fixed